tests/test_features.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
logger is unused, unless you want to keep it around for future enhancements

Reply:
removed